packages/std/src/query/wasm.rs

```diff
@@ -88,6 +146,39 @@ impl_hidden_constructor!(
 
 impl QueryResponseType for CodeInfoResponse {}
 
+#[non_exhaustive]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+pub struct RawRangeResponse {
+    /// The key-value pairs
+    pub data: Vec<RawRangeEntry>,
```

Comment:
Is there a reason to use the new `RawRangeEntry` instead of the existing `pub type Record` (which is a tuple instead of a struct)?

Reply:
I did that at first, but `Record` uses `Vec<u8>` for keys, instead of `Binary`, which is not ideal.

Reply:
Ah, I see. `Record` does not seem to be serde friendly in general. Or just inherits everything from the tuple because it is an alias.